### ISSUE TITLE
[Paperless] PAPERLESS_URL being set doesn't work with ingress

### ISSUE THREAD

**Issue by Emiller88** [k8s-at-home/charts#1545]
### Helm chart name

paperless

### Helm chart version

8.7.0

### Container name

ghcr.io/paperless-ngx/paperless-ngx

### Container tag

ngx-1.7.0

### Description

Introduced in https://github.com/k8s-at-home/charts/commit/12d8790717284534fb243c983e20851d7a0ef0dd#diff-a5bd3ab7ed2d41c4d2384082f4d38bb58b6e90cf2a31bc0bccd4022035d5c5f1=

https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security

> PAPERLESS_URL=<url>

>This setting can be used to set the three options below (ALLOWED_HOSTS, CORS_ALLOWED_HOSTS and CSRF_TRUSTED_ORIGINS). If the other options are set the values will be combined with this one. Do not include a trailing slash. E.g. https://paperless.domain.com/

Basically, there's no http/https in the front of the url.

### Expected result

The pod to statrt

### Helm values to reproduce

```YAML
`PAPERLESS_URL: "https://paperless.${SECRET_DOMAIN}"` solves it.
```

### Repo link

https://github.com/emiller88/home-cluster/commit/2e9fd5b

**Comment by angelnu** [comment]
Thanks @Emiller88 - I missed the http/https prefix.
